Services/src/main/java/ev/projects/services/ICaseService.java

```diff
@@ -6,14 +6,47 @@
 import java.util.List;
 import java.util.Optional;
 
+/**
+ * Business logic component for Case entity.
+ */
 @Service
 public interface ICaseService {
 
+    /**
+     * Eagerly retrieve cases.
+     * @return list of cases with each case containing list of documents.
+     */
     List<Case> getAllWithDocuments();
+
+    /**
+     * Lazily retrieve cases.
+     * @return list of cases.
+     */
     List<Case> getAll();
+
+    /**
```

Comment:
I did not add description to all methods because return type basically says what it does. For example `Optional<Case> getById(Long ID);` javadoc says: ` * @return Case, if exists by given ID.` Basically the javadoc return type description and method name say what it does. Should I still add a description? Because I would write something like: "gets a case by a given ID, if it exists" which is basically the same what is already written and kind of provides duplication, in my opinion.

Reply:
I have typed a comment in a place where it is difficult to read the code so here is the code:
`    /**
     * @param ID PK of case to be retrieved.
     * @return Case, if exists by given ID.
     */
    Optional<Case> getById(Long ID);`